app/src/main/java/mok/it/app/mokapp/feature/project_add_participants/AddOrRemoveMembersFragment.kt

```diff
@@ -0,0 +1,217 @@
+                                    Toast.LENGTH_SHORT
+                                )
+                                    .show()
+                                showDialog = DialogType.NONE
+                            },
+                            onDismiss = {
+                                showDialog = DialogType.NONE
+                            },
+                        )
+                    }
+
+                    DialogType.NONE -> {}
+                }
+                SearchField(
+                    searchQuery = searchQuery,
+                    chipState = chipState,
+                    onValueChange = { searchQuery = it },
+                )
+
+                LazyColumn {// selected users
+                    items(users.filter { uiState.selectedUserIds.contains(it.documentId) }) { user ->
+                        UserRow(
+                            user,
+                            true,
+                        ) { _ ->
+                            viewModel.userSelectionClicked(user)
+                        }
+                    }
+                }
+                HorizontalDivider(modifier = Modifier.padding(8.dp))
+                if (filteredUnselectedUsers.isNotEmpty()) {
+                    LazyColumn {// all users
+                        items(filteredUnselectedUsers) { user ->
+                            UserRow(
+                                user,
+                                false,
+                            ) { _ ->
+                                viewModel.userSelectionClicked(user)
+                            }
+                        }
+                    }
+                } else {
+                    Text(
+                        text = "Nincsenek a feltételeknek megfelelő tagok",
+                        modifier = Modifier.padding(16.dp),
+                        style = MaterialTheme.typography.headlineSmall,
+                        textAlign = androidx.compose.ui.text.style.TextAlign.Center
+                    )
+                }
+            }
+        }
+    }
+
+    @Composable
+    fun UserRow(
+        user: User,
+        isSelected: Boolean,
+        onCheckedChange: (Boolean) -> Unit,
+    ) {
+        Row(modifier = Modifier
+            .fillMaxWidth()
+            .height(48.dp)
+            .clickable {
+                onCheckedChange(!isSelected)
+            }
+            .padding(horizontal = 16.dp), verticalAlignment = Alignment.CenterVertically) {
+            UserIcon(user = user, navController = findNavController())
+            Text(
+                text = user.name,
+                style = MaterialTheme.typography.bodyLarge,
+                modifier = Modifier
+                    .padding(start = 16.dp)
+                    .weight(1f)
+            )
+            Checkbox(
+                checked = isSelected,
+                onCheckedChange = null // Accessibility: 'selectable' handles state
```

Comment:
Ez a sor mit jelent pontosan?

Reply:
azt, hogy direkt null az oncheckedchange-je a checkboxnak, mivel az egész soron van kezelve a kattintás

Reply:
Ez tök jó! Ilyenkor is lejátssza az animációt, ha a checkboxra kattint a user? Ez mondjuk tényleg apróság. Ha a kommentet kicsit átírnád, hogy ez értheyőbb legyen (pl "click is handled on whole row instead") az még segítene.

Reply:
asszem igen, le, de épp resolve-olom a többit, úgyhogy tesztelni nem tudom, de sztem igen :D Átírom a kommentet

Reply:
Köszönöm!